tests/test_object_creation.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from pyomeca import Analogs, Markers, Angles, Rototrans
+from ._constants import ANALOGS_DATA, MARKERS_DATA, EXPECTED_VALUES
+from .utils import is_expected_array
+
+
+def test_analogs_creation():
+    dims = ("channel", "time")
+    array = Analogs()
+    np.testing.assert_array_equal(x=array, y=xr.DataArray())
+    assert array.dims == dims
+
+    array = Analogs(ANALOGS_DATA.values)
+    is_expected_array(array, **EXPECTED_VALUES[56])
+
+    size = 10, 100
+    array = Analogs.from_random_data(size=size)
+    assert array.shape == size
+    assert array.dims == dims
+
+    with pytest.raises(ValueError):
+        Analogs(MARKERS_DATA)
+
+
+def test_markers_creation():
+    dims = ("axis", "channel", "time")
+    array = Markers()
+    np.testing.assert_array_equal(x=array, y=xr.DataArray())
+    assert array.dims == dims
+
+    array = Markers(MARKERS_DATA.values)
+    is_expected_array(array, **EXPECTED_VALUES[57])
+
+    size = 3, 10, 100
+    array = Markers.from_random_data(size=size)
```

Comment:
May I suggest to use a np.seed(42) so we always know what should be the outcome and test against this outcome

Reply:
It is already defined [here](https://github.com/romainmartinez/pyomeca/blob/897ad78c62848f85ea95e43f2f0587aa8009a9a0/tests/_constants.py#L15), which is executed before the tests are run